packages/biometric-ed25519/src/utils.ts

```diff
@@ -90,4 +90,28 @@ export const recoverPublicKey = async (r, s, message, recovery) => {
 export const uint8ArrayToBigInt = (uint8Array: Uint8Array) => {
     const array = Array.from(uint8Array);
     return BigInt('0x' + array.map(byte => byte.toString(16).padStart(2, '0')).join(''));
+};
+
+// This function is design to convert any existing Uint8Array properties inside AuthenticatorAssertionResponse to ArrayBuffer
+export const convertUint8ArrayToBuffer = (obj) => {
```

Comment:
Since this function converts `Uint8Array`, `Array` and `object` it's misleading to name it `convertUint8ArrayToBuffer`. Maybe `convertToArrayBuffer` is a better name
```suggestion
export const convertToArrayBuffer = (array) => {
```